modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/SequencesOnDisplay.scala

```diff
@@ -37,19 +36,35 @@ final case class SequencesOnDisplay(sequences: Zipper[SequenceTab]) {
   /**
    * List of loaded sequence ids
    */
-  val loadedIds: List[Observation.Id] =
-    sequences.toList.collect {
-      case InstrumentSequenceTab(_, curr, _, _) => curr().map(_.id)
-    }.collect {
-      case Some(x) => x
+  val loadedId: List[Observation.Id] =
```

Comment:
Not plural anymore?

Reply:
hmm I did that to facilitate the refactoring, it should be reverted back to plural